live_chat_adj/models/mail_channel.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+import re
+
+from odoo import api, models
+
+
+class MailChannel(models.Model):
+    _inherit = "mail.channel"
+
+    @api.multi
+    @api.returns("mail.message", lambda value: value.id)
+    def message_post(self, **kwargs):
+        list_value = kwargs.get("body").split()
+        keywords = re.findall(r"\w+", " ".join(list_value))
```

Comment:
Why we are not doing something like this `re.findall(r"\w+", kwargs.get("body"))`?